python/lsst/obs/hsc/hscMapper.py

```diff
@@ -164,13 +165,13 @@ def _flipChipsLR(exp, wcs, dataId, dims=None):
         return exp, wcs
 
     def std_raw_md(self, md, dataId):
-        if False:            # no std_raw_md in baseclass
-            md = super(HscMapper, self).std_raw_md(md, dataId)  # not present in baseclass
-        #
-        # We need to flip the WCS defined by the metadata in case anyone ever constructs a Wcs from it
-        #
+        """We need to flip the WCS defined by the metadata in case anyone ever
+        constructs a Wcs from it.
+        """
```

Comment:
Please add parameters/returns to docstring

Reply:
Ah, that reminds me.  @parejkoj, have a look at my adjustments to this commit here: https://github.com/lsst/obs_subaru/commit/95349380f894453f08bab564d4b70991880fd2d5